spec/items_orders_table.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY,
+ item_name text,
+ price int, 
+ quantity int
+);
+
+CREATE TABLE orders (
+  id SERIAL PRIMARY KEY,
+  customer_name text,
+  order_date text,
+  item_id int,
+  constraint fk_itemfo reign key(item_id)
+    references items(id)
+    on delete cascade
+);
+
+
+INSERT INTO items (item_name, price, quantity) 
```

Comment:
Possible way to refactor INSERT by writing only one INSERT statement and multiple values.